### PR TITLE
[server] Update node to 12.22.6 and add additional binaries

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Add additional binaries to help to understand the current state

## Related Issue(s)

Fixes #

## Release Notes

```release-note
[server] Update node to 12.22.6 and add additional binaries
```

/approve no-issue

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankoehnlein** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankoehnlein` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/server/leeway.Dockerfile [hunk 1]
```diff
@@ -12,11 +12,15 @@ WORKDIR /app
...
 
 
-FROM node:12.22.1-slim
+FROM node:12.22.6-slim
 
 # Using ssh-keygen for RSA keypair generation
 RUN apt-get update && apt-get install -yq \
         openssh-client \
+        procps \
+        net-tools \
```
Comment (geropl) [inline components/server/leeway.Dockerfile:21]: :+1:

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5776#issuecomment-925045598" title="Approved">JanKoehnlein</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5776#issuecomment-925045598" title="Approved">JanKoehnlein</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 03bdb79fe5d6d27bff6a5f8289f916e3799fc80f</details>

**Pull request merged**